reV/version.py

```diff
@@ -1,3 +1,5 @@
-"""reV Version number"""
+"""
+reV Version number
```

Comment:
You know you can rerun all checks on github right?

Reply:
NO!